cypress/integration/User.spec.ts

```diff
@@ -23,13 +12,12 @@ describe('Unauthenticated user', () => {
 
 describe('Authenticated user', () => {
   beforeEach(() => {
-    cy.visit(
-      `/target/${Cypress.env('target-id')}?${querystring.stringify(authObject)}`
-    );
+    cy.visitTargetPageAsAuthenticatedUser(Cypress.env('target-id'));
   });
 
   it('Has username in header', () => {
+    cy.get('header', { timeout: 5000 }).should('be.visible');
```

Comment:
I think the header might be there even when page is loading. If so, then it would be better to check element of the target page, like done in App.spec.ts

Reply:
It has to be some element that we have control of. I'll use the same element as in App.spec

Reply:
Fixed